docs/content/dagster-cloud/deployment/agents/running-multiple-agents.mdx

```diff
@@ -7,59 +7,89 @@ platform_type: "cloud"
 
 <Note>This guide is applicable to Dagster Cloud.</Note>
 
-Each Dagster Cloud full deployment (e.g., `prod`) needs to have at least one agent running. Since the agent is lightweight and can tolerate short downtimes such as during an upgrade, a single agent is adequate for most use cases.
+Each Dagster Cloud full deployment (e.g., `prod`) needs to have at least one agent running. A single agent is adequate for many use cases, but you may want to run multiple agents for the following reasons:
 
-If you want to distribute runs across multiple Kubernetes clusters, ECS clusters, EC2 instances, or other environments, you can run multiple agents. For example, one in each cluster, instance, etc.
+- Provide redundancy
+- Distribute runs across multiple Kubernetes clusters, ECS clusters, EC2 instances, or other environments
```

Comment:
This second bullet seems like the same thing as provide redundancy (unless they are different k8s clusters, which doesn't make sense unless you can route them to specific ones for some reason, which we don't support)

Maybe we ditch the bullet and say "but you may want to run multiple agents in order to provide redundancy if one agent goes down."

Reply:
I like this approach ^

Reply:
yup ended up going with this